### PR TITLE
Add Apache Mesos container and chart

### PR DESCRIPTION
**Pull request by algchoo**
* added container that has Apache Mesos installed
* added chart that includes a startup script for Apache Mesos
  * starts the master/agent/exporter all on the same pod

### CODE REVIEW

**Review by schmikei — COMMENTED** [review]
Works preem, just have one small question!

<details>
<summary> Metrics output from expo
```
# HELP go_gc_duration_seconds A summary of the GC invocation durations.
# TYPE go_gc_duration_seconds summary
go_gc_duration_seconds{quantile="0"} 0
go_gc_duration_seconds{quantile="0.25"} 0
go_gc_duration_seconds{quantile="0.5"} 0
go_gc_duration_seconds{quantile="0.75"} 0
go_gc_duration_seconds{quantile="1"} 0
go_gc_duration_seconds_sum 0
go_gc_duration_seconds_count 0
# HELP go_goroutines Number of goroutines that currently exist.
# TYPE go_goroutines gauge
go_goroutines 8
# HELP go_info Information about the Go environment.
# TYPE go_info gauge
go_info{version="go1.11.5"} 1
# HELP go_memstats_alloc_bytes Number of bytes allocated and still in use.
# TYPE go_memstats_alloc_bytes gauge
go_memstats_alloc_bytes 538472
# HELP go_memstats_alloc_bytes_total Total number of bytes allocated, even if freed.
# TYPE go_memstats_alloc_bytes_total counter
go_memstats_alloc_bytes_total 538472
# HELP go_memstats_buck_hash_sys_bytes Number of bytes used by the profiling bucket hash table.
# TYPE go_memstats_buck_hash_sys_bytes gauge
go_memstats_buck_hash_sys_bytes 1.442966e+06
# HELP go_memstats_frees_total Total number of frees.
# TYPE go_memstats_frees_total counter
go_memstats_frees_total 173
# HELP go_memstats_gc_cpu_fraction The fraction of this program's available CPU time used by the GC since the program started.
# TYPE go_memstats_gc_cpu_fraction gauge
go_memstats_gc_cpu_fraction 0
# HELP go_memstats_gc_sys_bytes Number of bytes used for garbage collection system metadata.
# TYPE go_memstats_gc_sys_bytes gauge
go_memstats_gc_sys_bytes 2.234368e+06
# HELP go_memstats_heap_alloc_bytes Number of heap bytes allocated and still in use.
# TYPE go_memstats_heap_alloc_bytes gauge
go_memstats_heap_alloc_bytes 538472
# HELP go_memstats_heap_idle_bytes Number of heap bytes waiting to be used.
# TYPE go_memstats_heap_idle_bytes gauge
go_memstats_heap_idle_bytes 6.4536576e+07
# HELP go_memstats_heap_inuse_bytes Number of heap bytes that are in use.
# TYPE go_memstats_heap_inuse_bytes gauge
go_memstats_heap_inuse_bytes 1.982464e+06
# HELP go_memstats_heap_objects Number of allocated objects.
# TYPE go_memstats_heap_objects gauge
go_memstats_heap_objects 2633
# HELP go_memstats_heap_released_bytes Number of heap bytes released to OS.
# TYPE go_memstats_heap_released_bytes gauge
go_memstats_heap_released_bytes 0
# HELP go_memstats_heap_sys_bytes Number of heap bytes obtained from system.
# TYPE go_memstats_heap_sys_bytes gauge
go_memstats_heap_sys_bytes 6.651904e+07
# HELP go_memstats_last_gc_time_seconds Number of seconds since 1970 of last garbage collection.
# TYPE go_memstats_last_gc_time_seconds gauge
go_memstats_last_gc_time_seconds 0
# HELP go_memstats_lookups_total Total number of pointer lookups.
# TYPE go_memstats_lookups_total counter
go_memstats_lookups_total 0
# HELP go_memstats_mallocs_total Total number of mallocs.
# TYPE go_memstats_mallocs_total counter
go_memstats_mallocs_total 2806
# HELP go_memstats_mcache_inuse_bytes Number of bytes in use by mcache structures.
# TYPE go_memstats_mcache_inuse_bytes gauge
go_memstats_mcache_inuse_bytes 12096
# HELP go_memstats_mcache_sys_bytes Number of bytes used for mcache structures obtained from system.
# TYPE go_memstats_mcache_sys_bytes gauge
go_memstats_mcache_sys_bytes 16384
# HELP go_memstats_mspan_inuse_bytes Number of bytes in use by mspan structures.
# TYPE go_memstats_mspan_inuse_bytes gauge
go_memstats_mspan_inuse_bytes 31768
# HELP go_memstats_mspan_sys_bytes Number of bytes used for mspan structures obtained from system.
# TYPE go_memstats_mspan_sys_bytes gauge
go_memstats_mspan_sys_bytes 32768
# HELP go_memstats_next_gc_bytes Number of heap bytes when next garbage collection will take place.
# TYPE go_memstats_next_gc_bytes gauge
go_memstats_next_gc_bytes 4.473924e+06
# HELP go_memstats_other_sys_bytes Number of bytes used for other system allocations.
# TYPE go_memstats_other_sys_bytes gauge
go_memstats_other_sys_bytes 1.055842e+06
# HELP go_memstats_stack_inuse_bytes Number of bytes in use by the stack allocator.
# TYPE go_memstats_stack_inuse_bytes gauge
go_memstats_stack_inuse_bytes 589824
# HELP go_memstats_stack_sys_bytes Number of bytes obtained from system for stack allocator.
# TYPE go_memstats_stack_sys_bytes gauge
go_memstats_stack_sys_bytes 589824
# HELP go_memstats_sys_bytes Number of bytes obtained from system.
# TYPE go_memstats_sys_bytes gauge
go_memstats_sys_bytes 7.1891192e+07
# HELP go_threads Number of OS threads created.
# TYPE go_threads gauge
go_threads 11
# HELP mesos_collector_errors_total Total number of internal mesos-collector errors.
# TYPE mesos_collector_errors_total counter
mesos_collector_errors_total 0
# HELP mesos_exporter_build_info A metric with a constant '1' value labeled by version, revision, branch, and goversion from which mesos_exporter was built.
# TYPE mesos_exporter_build_info gauge
mesos_exporter_build_info{branch="master",goversion="go1.11.5",revision="ba95fe9b277e7940db33ee62c25e7b1709ec28cb",version="1.1.2"} 1
# HELP mesos_master_allocation_run_latency_ms Allocation batch latency in ms.
# TYPE mesos_master_allocation_run_latency_ms gauge
mesos_master_allocation_run_latency_ms{type="max"} 38.683136
mesos_master_allocation_run_latency_ms{type="mean"} 0.491776
mesos_master_allocation_run_latency_ms{type="min"} 0.34688
mesos_master_allocation_run_latency_ms{type="p50"} 0.564224
mesos_master_allocation_run_latency_ms{type="p90"} 0.997888
mesos_master_allocation_run_latency_ms{type="p95"} 2.710784
mesos_master_allocation_run_latency_ms{type="p99"} 16.437350400000078
mesos_master_allocation_run_latency_ms{type="p999"} 36.45855744000005
mesos_master_allocation_run_latency_ms{type="p9999"} 38.46067814400012
# HELP mesos_master_allocation_run_latency_ms_count Number of allocation batch latency measurements
# TYPE mesos_master_allocation_run_latency_ms_count counter
mesos_master_allocation_run_latency_ms_count{event="allocation"} 81
# HELP mesos_master_allocation_run_ms Time spent in allocation algorithm in ms.
# TYPE mesos_master_allocation_run_ms gauge
mesos_master_allocation_run_ms{type="max"} 93.411072
mesos_master_allocation_run_ms{type="mean"} 0.545792
mesos_master_allocation_run_ms{type="min"} 0.150016
mesos_master_allocation_run_ms{type="p50"} 0.572928
mesos_master_allocation_run_ms{type="p90"} 0.900096
mesos_master_allocation_run_ms{type="p95"} 1.321984
mesos_master_allocation_run_ms{type="p99"} 38.3239680000002
mesos_master_allocation_run_ms{type="p999"} 87.90236160000012
mesos_master_allocation_run_ms{type="p9999"} 92.86020096000031
# HELP mesos_master_allocation_run_ms_count Number of allocation algorithm time measurements in the window
# TYPE mesos_master_allocation_run_ms_count gauge
mesos_master_allocation_run_ms_count 81
# HELP mesos_master_allocation_runs Number of times the allocation alorithm has run
# TYPE mesos_master_allocation_runs counter
mesos_master_allocation_runs{event="allocation"} 81
# HELP mesos_master_allocator_event_queue_dispatches Number of dispatch events in the allocator event queue.
# TYPE mesos_master_allocator_event_queue_dispatches gauge
mesos_master_allocator_event_queue_dispatches 0
# HELP mesos_master_allocator_resources_cpus Number of CPUs offered or allocated
# TYPE mesos_master_allocator_resources_cpus gauge
mesos_master_allocator_resources_cpus{type="offered_or_allocated"} 0
mesos_master_allocator_resources_cpus{type="total"} 7
# HELP mesos_master_allocator_resources_disk Allocated or offered disk space in MB
# TYPE mesos_master_allocator_resources_disk gauge
mesos_master_allocator_resources_disk{type="offered_or_allocated"} 0
mesos_master_allocator_resources_disk{type="total"} 54699
# HELP mesos_master_allocator_resources_mem Allocated or offered memory in MB
# TYPE mesos_master_allocator_resources_mem gauge
mesos_master_allocator_resources_mem{type="offered_or_allocated"} 0
mesos_master_allocator_resources_mem{type="total"} 10955
# HELP mesos_master_cpus Current CPU resources in cluster.
# TYPE mesos_master_cpus gauge
mesos_master_cpus{type="free"} 7
mesos_master_cpus{type="percent"} 0
mesos_master_cpus{type="total"} 7
mesos_master_cpus{type="used"} 0
# HELP mesos_master_cpus_revocable Current revocable CPU resources in cluster.
# TYPE mesos_master_cpus_revocable gauge
mesos_master_cpus_revocable{type="free"} 0
mesos_master_cpus_revocable{type="percent"} 0
mesos_master_cpus_revocable{type="total"} 0
mesos_master_cpus_revocable{type="used"} 0
# HELP mesos_master_disk Current disk resources in cluster.
# TYPE mesos_master_disk gauge
mesos_master_disk{type="free"} 54699
mesos_master_disk{type="percent"} 0
mesos_master_disk{type="total"} 54699
mesos_master_disk{type="used"} 0
# HELP mesos_master_disk_revocable Current disk resources in cluster.
# TYPE mesos_master_disk_revocable gauge
mesos_master_disk_revocable{type="free"} 0
mesos_master_disk_revocable{type="percent"} 0
mesos_master_disk_revocable{type="total"} 0
mesos_master_disk_revocable{type="used"} 0
# HELP mesos_master_elected 1 if master is elected leader, 0 if not
# TYPE mesos_master_elected gauge
mesos_master_elected 1
# HELP mesos_master_event_queue_dispatches Number of dispatch events in the allocator mesos event queue.
# TYPE mesos_master_event_queue_dispatches gauge
mesos_master_event_queue_dispatches 0
# HELP mesos_master_event_queue_length Current number of elements in event queue by type
# TYPE mesos_master_event_queue_length gauge
mesos_master_event_queue_length{type="dispatches"} 30
mesos_master_event_queue_length{type="http_request"} 0
mesos_master_event_queue_length{type="message"} 0
# HELP mesos_master_frameworks_state Current number of frames known to the master per connection and registration state.
# TYPE mesos_master_frameworks_state gauge
mesos_master_frameworks_state{state="connected_active"} 0
mesos_master_frameworks_state{state="connected_inactive"} 0
mesos_master_frameworks_state{state="disconnected_inactive"} 0
# HELP mesos_master_gpus Current GPU resources in cluster.
# TYPE mesos_master_gpus gauge
mesos_master_gpus{type="free"} 0
mesos_master_gpus{type="percent"} 0
mesos_master_gpus{type="total"} 0
mesos_master_gpus{type="used"} 0
# HELP mesos_master_gpus_revocable Current revocable GPU resources in cluster.
# TYPE mesos_master_gpus_revocable gauge
mesos_master_gpus_revocable{type="free"} 0
mesos_master_gpus_revocable{type="percent"} 0
mesos_master_gpus_revocable{type="total"} 0
mesos_master_gpus_revocable{type="used"} 0
# HELP mesos_master_mem Current memory resources in cluster.
# TYPE mesos_master_mem gauge
mesos_master_mem{type="free"} 10955
mesos_master_mem{type="percent"} 0
mesos_master_mem{type="total"} 10955
mesos_master_mem{type="used"} 0
# HELP mesos_master_mem_revocable Current revocable memory resources in cluster.
# TYPE mesos_master_mem_revocable gauge
mesos_master_mem_revocable{type="free"} 0
mesos_master_mem_revocable{type="percent"} 0
mesos_master_mem_revocable{type="total"} 0
mesos_master_mem_revocable{type="used"} 0
# HELP mesos_master_messages Number of messages by the master by state
# TYPE mesos_master_messages counter
mesos_master_messages{type="authenticate_messages"} 0
mesos_master_messages{type="deactivate_framework"} 0
mesos_master_messages{type="decline_offers"} 0
mesos_master_messages{type="dropped_messages"} 2
mesos_master_messages{type="executor_to_framework"} 0
mesos_master_messages{type="exited_executor"} 0
mesos_master_messages{type="framework_to_executor"} 0
mesos_master_messages{type="kill_task"} 0
mesos_master_messages{type="launch_tasks"} 0
mesos_master_messages{type="reconcile_tasks"} 0
mesos_master_messages{type="register_framework"} 0
mesos_master_messages{type="register_slave"} 5
mesos_master_messages{type="reregister_framework"} 0
mesos_master_messages{type="reregister_slave"} 0
mesos_master_messages{type="resource_request"} 0
mesos_master_messages{type="revive_offers"} 0
mesos_master_messages{type="status_update"} 0
mesos_master_messages{type="status_update_acknowledgement"} 0
mesos_master_messages{type="suppress_offers"} 0
mesos_master_messages{type="unregister_framework"} 0
mesos_master_messages{type="unregister_slave"} 0
mesos_master_messages{type="update_slave"} 0
# HELP mesos_master_messages_outcomes_total Total number of messages by outcome of operation and direction.
# TYPE mesos_master_messages_outcomes_total counter
mesos_master_messages_outcomes_total{destination="executor",outcome="invalid",source="framework",type=""} 0
mesos_master_messages_outcomes_total{destination="executor",outcome="valid",source="framework",type=""} 0
mesos_master_messages_outcomes_total{destination="framework",outcome="invalid",source="executor",type=""} 0
mesos_master_messages_outcomes_total{destination="framework",outcome="invalid",source="slave",type="status_update"} 0
mesos_master_messages_outcomes_total{destination="framework",outcome="valid",source="executor",type=""} 0
mesos_master_messages_outcomes_total{destination="framework",outcome="valid",source="slave",type="status_update"} 0
mesos_master_messages_outcomes_total{destination="slave",outcome="invalid",source="framework",type="status_update"} 0
mesos_master_messages_outcomes_total{destination="slave",outcome="valid",source="framework",type="status_update"} 0
# HELP mesos_master_offers_pending Current number of offers made by the master which aren't yet accepted or declined by frameworks.
# TYPE mesos_master_offers_pending gauge
mesos_master_offers_pending 0
# HELP mesos_master_recovery_slave_removal_events_total Total number of recovery removal events on this master since it booted.
# TYPE mesos_master_recovery_slave_removal_events_total counter
mesos_master_recovery_slave_removal_events_total{event="removal"} 0
# HELP mesos_master_slave_registration_events_total Total number of registration events on this master since it booted.
# TYPE mesos_master_slave_registration_events_total counter
mesos_master_slave_registration_events_total{event="register"} 1
mesos_master_slave_registration_events_total{event="reregister"} 0
# HELP mesos_master_slave_removal_events_reasons Total number of slave removal events by reason on this master since it booted.
# TYPE mesos_master_slave_removal_events_reasons counter
mesos_master_slave_removal_events_reasons{reason="registered"} 0
mesos_master_slave_removal_events_reasons{reason="unhealthy"} 0
mesos_master_slave_removal_events_reasons{reason="unregistered"} 0
# HELP mesos_master_slave_removal_events_total Total number of removal events on this master since it booted.
# TYPE mesos_master_slave_removal_events_total counter
mesos_master_slave_removal_events_total{event="canceled"} 0
mesos_master_slave_removal_events_total{event="completed"} 0
mesos_master_slave_removal_events_total{event="died"} 0
mesos_master_slave_removal_events_total{event="scheduled"} 0
# HELP mesos_master_slave_unreachable_events_total Total number of slave unreachable events on this master since it booted.
# TYPE mesos_master_slave_unreachable_events_total counter
mesos_master_slave_unreachable_events_total{event="canceled"} 0
mesos_master_slave_unreachable_events_total{event="completed"} 0
mesos_master_slave_unreachable_events_total{event="scheduled"} 0
# HELP mesos_master_slaves_state Current number of slaves known to the master per connection and registration state.
# TYPE mesos_master_slaves_state gauge
mesos_master_slaves_state{state="connected_active"} 1
mesos_master_slaves_state{state="connected_inactive"} 0
mesos_master_slaves_state{state="disconnected_inactive"} 0
mesos_master_slaves_state{state="unreachable"} 0
# HELP mesos_master_task_states_current Current number of tasks by state.
# TYPE mesos_master_task_states_current gauge
mesos_master_task_states_current{state="killing"} 0
mesos_master_task_states_current{state="running"} 0
mesos_master_task_states_current{state="staging"} 0
mesos_master_task_states_current{state="starting"} 0
mesos_master_task_states_current{state="unreachable"} 0
# HELP mesos_master_task_states_exit_total Total number of tasks processed by exit state.
# TYPE mesos_master_task_states_exit_total counter
mesos_master_task_states_exit_total{state="dropped"} 0
mesos_master_task_states_exit_total{state="errored"} 0
mesos_master_task_states_exit_total{state="failed"} 0
mesos_master_task_states_exit_total{state="finished"} 0
mesos_master_task_states_exit_total{state="gone"} 0
mesos_master_task_states_exit_total{state="gone_by_operator"} 0
mesos_master_task_states_exit_total{state="killed"} 0
mesos_master_task_states_exit_total{state="lost"} 0
# HELP mesos_master_uptime_seconds Number of seconds the master process is running.
# TYPE mesos_master_uptime_seconds gauge
mesos_master_uptime_seconds 81.92638592
# HELP mesos_overlay_log_ensemble_size Ensemble size of the overlay log
# TYPE mesos_overlay_log_ensemble_size gauge
mesos_overlay_log_ensemble_size 0
# HELP mesos_overlay_log_recovered Recovered status of the overlay log
# TYPE mesos_overlay_log_recovered gauge
mesos_overlay_log_recovered 0
# HELP mesos_registrar_log_ensemble_size Ensemble size of the registrar log
# TYPE mesos_registrar_log_ensemble_size gauge
mesos_registrar_log_ensemble_size 1
# HELP mesos_registrar_log_recovered Recovered status of the registrar log
# TYPE mesos_registrar_log_recovered gauge
mesos_registrar_log_recovered 1
# HELP mesos_registrar_queued_operations Number of operations in the registry queue
# TYPE mesos_registrar_queued_operations gauge
mesos_registrar_queued_operations 0
# HELP mesos_registrar_registry_size_bytes Size of the registry in bytes
# TYPE mesos_registrar_registry_size_bytes gauge
mesos_registrar_registry_size_bytes 381
# HELP mesos_registrar_state_fetch_ms Duration of state JSON fetch in ms
# TYPE mesos_registrar_state_fetch_ms gauge
mesos_registrar_state_fetch_ms 1263.320064
# HELP mesos_registrar_state_store_ms Duration of state json store in ms.
# TYPE mesos_registrar_state_store_ms gauge
mesos_registrar_state_store_ms{type="max"} 196.864256
mesos_registrar_state_store_ms{type="mean"} 103.858944
mesos_registrar_state_store_ms{type="min"} 103.858944
mesos_registrar_state_store_ms{type="p50"} 150.3616
mesos_registrar_state_store_ms{type="p90"} 187.56372480000002
mesos_registrar_state_store_ms{type="p95"} 192.2139904
mesos_registrar_state_store_ms{type="p99"} 195.93420288000002
mesos_registrar_state_store_ms{type="p999"} 196.771250688
mesos_registrar_state_store_ms{type="p9999"} 196.85495546880003
# HELP mesos_slave_cpus Total slave CPUs (fractional)
# TYPE mesos_slave_cpus gauge
mesos_slave_cpus{slave="slave(1)@172.23.0.3:5051"} 7
# HELP mesos_slave_cpus_unreserved Unreserved slave CPUs (fractional)
# TYPE mesos_slave_cpus_unreserved gauge
mesos_slave_cpus_unreserved{slave="slave(1)@172.23.0.3:5051"} 7
# HELP mesos_slave_cpus_used Used slave CPUs (fractional)
# TYPE mesos_slave_cpus_used gauge
mesos_slave_cpus_used{slave="slave(1)@172.23.0.3:5051"} 0
# HELP mesos_slave_disk_bytes Total slave disk space in bytes
# TYPE mesos_slave_disk_bytes gauge
mesos_slave_disk_bytes{slave="slave(1)@172.23.0.3:5051"} 5.6011776e+07
# HELP mesos_slave_disk_unreserved_bytes Unreserved slave disk in bytes
# TYPE mesos_slave_disk_unreserved_bytes gauge
mesos_slave_disk_unreserved_bytes{slave="slave(1)@172.23.0.3:5051"} 5.6011776e+07
# HELP mesos_slave_disk_used_bytes Used slave disk space in bytes
# TYPE mesos_slave_disk_used_bytes gauge
mesos_slave_disk_used_bytes{slave="slave(1)@172.23.0.3:5051"} 0
# HELP mesos_slave_mem_bytes Total slave memory in bytes
# TYPE mesos_slave_mem_bytes gauge
mesos_slave_mem_bytes{slave="slave(1)@172.23.0.3:5051"} 1.121792e+07
# HELP mesos_slave_mem_unreserved_bytes Unreserved slave memory in bytes
# TYPE mesos_slave_mem_unreserved_bytes gauge
mesos_slave_mem_unreserved_bytes{slave="slave(1)@172.23.0.3:5051"} 1.121792e+07
# HELP mesos_slave_mem_used_bytes Used slave memory in bytes
# TYPE mesos_slave_mem_used_bytes gauge
mesos_slave_mem_used_bytes{slave="slave(1)@172.23.0.3:5051"} 0
# HELP mesos_slave_ports Total slave ports
# TYPE mesos_slave_ports gauge
mesos_slave_ports{slave="slave(1)@172.23.0.3:5051"} 1001
# HELP mesos_slave_ports_unreserved Unreserved slave ports
# TYPE mesos_slave_ports_unreserved gauge
mesos_slave_ports_unreserved{slave="slave(1)@172.23.0.3:5051"} 1001
# HELP mesos_slave_ports_used Used slave ports
# TYPE mesos_slave_ports_used gauge
mesos_slave_ports_used{slave="slave(1)@172.23.0.3:5051"} 0
# HELP process_cpu_seconds_total Total user and system CPU time spent in seconds.
# TYPE process_cpu_seconds_total counter
process_cpu_seconds_total 0
# HELP process_max_fds Maximum number of open file descriptors.
# TYPE process_max_fds gauge
process_max_fds 1.048576e+06
# HELP process_open_fds Number of open file descriptors.
# TYPE process_open_fds gauge
process_open_fds 8
# HELP process_resident_memory_bytes Resident memory size in bytes.
# TYPE process_resident_memory_bytes gauge
process_resident_memory_bytes 0
# HELP process_start_time_seconds Start time of the process since unix epoch in seconds.
# TYPE process_start_time_seconds gauge
process_start_time_seconds 1.685365844e+09
# HELP process_virtual_memory_bytes Virtual memory size in bytes.
# TYPE process_virtual_memory_bytes gauge
process_virtual_memory_bytes 0

```

File: charts/mesos/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      - name: mesos
+        image: {{ .Values.image }}
+        env:
+        - name: LD_LIBRARY_PATH
+          value: "/usr/local/lib"
+        ports:
+        - containerPort: 5050
+          name: mesos-master
+        - containerPort: 5051
+          name: mesos-agent
```
[the remark this answers is not in the document]
Comment (schmikei) [reply charts/mesos/templates/deployment.yaml:26]: Discussed with Austin, think we should specify a service for this as well

File: charts/apache-cassandra/Chart.yaml
Comment (schmikei) [inline charts/apache-cassandra/Chart.yaml:1]: We shouldnt change this one right for backwards compatibility and we won't have to update any dependents like cloud-onboarding?

Comment (algchoo) [reply charts/apache-cassandra/Chart.yaml:1]: I wasn't sure, but I will change it back, thanks!

File: charts/apache-mesos/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
+appVersion: tbd
+engine: gotpl
+home: https://github.com/observIQ/charts/tree/main/charts/mesos
```
Comment (schmikei) [inline charts/apache-mesos/Chart.yaml:4]:
```suggestion
home: https://github.com/observIQ/charts/tree/main/charts/apache-mesos
```

**Review by schmikei — APPROVED** [review]
Small gotcha but overall runs good!

**Pull request merged**